app/components/admin/users/tools/credits_component.html.erb

```diff
@@ -0,0 +1,144 @@
+                class: "my-3",
+                id: :credits_add do |f| %>
+    <fieldset>
+      <legend class="fs-base fw-medium mb-3"><%= pluralize(@user_unspent_credits_count, "available user credit") %></legend>
+
+      <div class="crayons-field wb-2">
+        <%= f.label "Add credits", class: "crayons-field__label", id: "add-credits" %>
+      </div>
+
+      <div class="crayons-field form-group w-50">
+        <%= f.number_field "credits[count]",
+                           id: nil,
+                           in: 1...10_000,
+                           required: true,
+                           class: "crayons-textfield",
+                           "aria-labelledby": "add-credits" %>
+        <%= f.text_field "credits[note]",
+                         id: nil,
+                         placeholder: "Why are you adding these credits?",
+                         required: true,
+                         class: "crayons-textfield",
+                         "aria-labelledby": "add-credits" %>
+      </div>
+
+      <%= f.submit "Add", class: "crayons-btn", data: { disable_with: "Add" } %>
+    </fieldset>
+  <% end -%>
+
+  <%# Remove user credits %>
+  <% if @user_unspent_credits_count.positive? %>
+    <%= form_with url: admin_user_tools_credits_path(@user),
+                  method: :delete,
+                  remote: true,
+                  class: "my-3",
+                  id: :credits_remove do |f| %>
+      <fieldset>
+        <div class="crayons-field mb-2">
+          <%= f.label "Remove credits", class: "crayons-field__label", id: "remove-credits" %>
+        </div>
+
+        <div class="crayons-field form-group w-50">
+          <%= f.number_field "credits[count]",
+                             id: nil,
+                             in: 1..@user_unspent_credits_count,
+                             required: true,
+                             class: "crayons-textfield mr-1",
+                             "aria-labelledby": "remove-credits" %>
+          <%= f.text_field "credits[note]",
+                           id: nil,
+                           placeholder: "Why are you removing these credits?",
+                           required: true,
+                           class: "crayons-textfield mr-1",
+                           "aria-labelledby": "remove-credits" %>
+        </div>
+
+        <%= f.submit "Remove", class: "crayons-btn crayons-btn--danger", data: { disable_with: "Remove" } %>
+      </fieldset>
+    <% end -%>
+  <% end %>
+
+  <%# Add and remove organizations credits %>
+  <% if @organizations.any? %>
+    <%= form_with url: admin_user_tools_credits_path(@user),
+                  method: :post,
+                  remote: true,
+                  class: "my-3",
+                  id: :org_credits_add do |f| %>
+
+      <fieldset>
+        <div class="crayons-field mb-1">
+          <%= f.label "Add credits to organizations", class: "crayons-field__label", id: "add-org-credits" %>
```

Comment:
Same as above, this should be the `legend` and the inputs below should have aria-labels. Same for the "remove credits" section below

Reply:
thanks!